recipes/draco/all/conanfile.py

```diff
@@ -49,8 +49,8 @@ def config_options(self):
             del self.options.fPIC
 
     def configure(self):
-        if self.options.shared:
-            del self.options.fPIC
```

Comment:
For my knowledge, why was this removed?

Reply:
disabling fPIC for shared library is an error, that's why it's removed. The only motivation to keep it for both, is because fPIC is a default option just like shared, and removing it could break users which force fPIC=False by default (we saw it in the past running Conan Package Tools).

Reply:
Removing that `fPIC` option does not disable fpic. It's just keeps the default.
https://github.com/conan-io/conan/blob/f361690beae3608d93a8d79dec7abb24f75ddd85/conans/client/build/cmake_flags.py#L337-L343
When settings `shared=True` and `fPIC=False`, a build will fail if a shared library includes a static library (that has not overridden the pic flag).

Reply:
Ahhh thank you both ! I am pretty sure a recipe I merged has this